src/main.c

```diff
@@ -69,17 +69,51 @@ void connect_console_command(char *str) {
         g->typing = 1;
         g->typing_buffer[0] = '\0';
     } else if (g->server_user[0] == '\0') {
-        strncpy(g->server_user, str, 32);
-        snprintf(g->text_message, KONSTRUCTS_TEXT_MESSAGE_SIZE,
-                "Enter a password to login/create account");
-        snprintf(g->text_prompt, KONSTRUCTS_TEXT_MESSAGE_SIZE, "Password");
+        char new_str[strlen(str)+1]; // Strip all whitespaces away
```

Comment:
What would you say about `char new_str[MAX(32,strlen(str)+1)]`? Might be a bit hacky but should prevent overflow well